### DISCUSSION TITLE
vsxmake不会把add_tests的runargs带过去

### DISCUSSION THREAD

**Discussion by Elite-stay** [xmake-io/xmake#5223] [Q&A]
`add_tests("\b", {runargs = {"aaa", "bbb"}})`这样的不行，得写成`set_runargs`的

**Comment by waruqi** [comment]
不懂你在说啥

**Reply by Elite-stay** [reply]
就是用xmake project -k vsxmake生成的sln打开之后调试里的启动参数是$(XmakeRunArgs)，不是add_tests的runargs里设置的，这样到vs里F5就没有参数，要改成set_runargs才行

![图片](https://github.com/xmake-io/xmake/assets/9249009/ed2883bb-5157-42d1-88dc-1d0eccdc4587)

**Reply by Elite-stay** [reply]
或者说，如果我想在vs和linux里调试test类型的目标该怎么做呢

**Reply by waruqi** [reply]
不要直接调 tests，不支持，改成常规 target 走 run 调试
